tests/fixtures/parsers/space-before-function-paren/function-type-annotation.js

```diff
@@ -0,0 +1,525 @@
+"use strict";
```

Comment:
I'm not sure, but would it make sense to name the folder `tests/fixtures/parsers/babel/` instead? Or maybe we could clean up/re-organize the `fixtures/parsers/` folder in a separate PR. I think it's a bit confusing now where to put parser fixtures.. related: https://github.com/eslint/eslint/pull/8297/files#diff-824f19921586246afb6f381021c34222